### PR TITLE
Completed Find First and Last Index- Please Review

### PR DESCRIPTION
**Pull request by SandhyaGitH**

### CODE REVIEW

File: Sample.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
+// Time Complexity :O(logn)
```
Comment (rimzimt) [inline Sample.java:1]: define n

File: Sample.java [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+                 end = mid-1;
+               // go on left side 
+            }
+           
+         }
+        
+        return result;
+    }
+}
+
```
Comment (rimzimt) [inline Sample.java:65]: Good job!

**Review by rimzimt — COMMENTED** [review]
Keep it up Sandhya!

**Comment by rimzimt** [comment]
Keep it up Sandhya!

**Comment by SandhyaGitH** [comment]
close

**Pull request closed without merging**